### PR TITLE
Updated readme to be more accurate and provide more information

### PR DESCRIPTION
**Pull request by malscent**

### CODE REVIEW

File: marketplace/createUiDefinition.json [hunk 1]
```diff
@@ -267,10 +267,14 @@
             "name": "serverVersion",
             "type": "Microsoft.Common.DropDown",
             "label": "Server Version",
-            "defaultValue": "6.5.1",
+            "defaultValue": "6.6.0",
```
Comment (ingenthr) [inline marketplace/createUiDefinition.json]: Given that 6.6.1 is out, I'd think we should default to that.

File: marketplace/createUiDefinition.json [hunk 2]
```diff
@@ -267,10 +267,14 @@
...
             "toolTip": "Server Version",
             "constraints": {
               "allowedValues": [
+                {
```
Comment (ingenthr) [inline marketplace/createUiDefinition.json:274]: 6.6.1 should be in here as well, right?

File: marketplace/mainTemplate-byol_2019.json [hunk 3]
```diff
@@ -291,7 +291,7 @@
               "publisher": "couchbase",
               "offer": "couchbase-server-enterprise",
               "sku": "[parameters('license')]",
-              "version": "3.0.2"
+              "version": "18.4.0"
```
Comment (tahmmee) [inline marketplace/mainTemplate-byol_2019.json]: I was expecting this to be 6.6.0 or some kind of Couchbase version, but I'm assuming the previous versioning was wrong here, as it is now being used to represent the base vm version.

File: stack/deploy.sh [hunk 4]
```diff
@@ -1,7 +1,60 @@
 #!/bin/sh
```
Comment (ingenthr) [inline stack/deploy.sh:1]: not your bug, but this looks like it relies on bashisms, not simple bourne shell.

Comment (ingenthr) [reply stack/deploy.sh:1]: maybe run it past a linter somewhere?

File: stack/mainTemplate.json [hunk 5]
```diff
@@ -1,47 +1,148 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
+    /* The number of nodes to deploy */
```
Comment (ingenthr) [inline stack/mainTemplate.json:5]: love the comments!  however, doesnn't that make it invalid JSON?  This is some extended JSON?

**Review by tahmmee — COMMENTED** [review]
Great job picking this up Justin! Very minor comment/question

File: stack/mainTemplate.json [hunk 6]
```diff
@@ -348,23 +480,28 @@
...
           ]
         }
       }
     },
     {
-      "apiVersion": "2016-03-30",
+      "apiVersion": "2019-07-01",
       "type": "Microsoft.Compute/virtualMachines/extensions",
-      "name": "[concat('server', copyindex(), '/install')]",
+      /*For troupbleshooting the script being executed see https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/custom-script-linux#troubleshooting
```
Comment (ingenthr) [inline stack/mainTemplate.json]: I've never troupbleshooted before.  :D

**Review by ingenthr — CHANGES_REQUESTED** [review]
The only real change we need for sure is 6.6.1 I think.  Of course, if you intentionally left that out, it's fine.

File: marketplace/mainTemplate-byol_2019.json (hunk 3 above)
Comment (malscent) [reply marketplace/mainTemplate-byol_2019.json]: Yeah.. This is the base VM image version, not the couchbase version

File: stack/mainTemplate.json (hunk 5 above)
Comment (malscent) [reply stack/mainTemplate.json:5]: Yeah ARM templates are an extended JSON format

File: stack/deploy.sh (hunk 4 above)
Comment (malscent) [reply stack/deploy.sh:1]: done!

**Comment by tahmmee** [comment]
lgtm

**Review by ingenthr — APPROVED** [review]

**Comment by ingenthr** [comment]
Note that I cherry picked out the actual changes from under the merge commit.  I also fixed attribution.  Note @malscent, your author email was slightly off.  Not sure why.  But I just got the needed bits out of the merge commit for a ffwd commit.

All merged now.

**Comment by malscent** [comment]
Doh..

I’ll fix that author email ASAP.

Thanks

Justin Ashworth
Senior Software Engineer‌
justin.ashworth@couchbase.com‌
‌

[v63+PdS7y8wL78vdOPJZY8LLXUfwHZLxH6vUBIYgAAAABJRU5ErkJggg==]

**Pull request closed without merging**